cmd/buildah/run.go

```diff
@@ -51,6 +51,9 @@ func runCmd(c *cli.Context) error {
 	}
 	name := args[0]
 	args = args.Tail()
+	if len(args) > 0 && args[0] == "--" {
```

Comment:
"--" is the CLI standard.  And it built into the CLI parsing in GOLANG.
https://unix.stackexchange.com/questions/55753/how-to-deal-with-end-of-options-in-getopts